server/http.go

```diff
@@ -183,55 +189,42 @@ func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if !strings.Contains(r.Header.Get("Content-Type"), "application/json") {
-		res := fmt.Sprintf("Expected Content-Type 'application/json' for webhook request, received '%s'.", r.Header.Get("Content-Type"))
-		http.Error(w, res, http.StatusBadRequest)
-		return
-	}
-
-	b, err := ioutil.ReadAll(r.Body)
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusBadRequest)
+	if err := r.ParseForm(); err != nil {
+		http.Error(w, "Bad request body", http.StatusBadRequest)
 		return
 	}
 
 	var webhook zoom.Webhook
-	err = json.Unmarshal(b, &webhook)
-	if err != nil {
-		p.API.LogError("Error unmarshaling webhook", "err", err.Error())
+	decoder := schema.NewDecoder()
+
+	// Try to decode to standard webhook
+	if err := decoder.Decode(&webhook, r.PostForm); err != nil {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
 
-	if webhook.Event != zoom.EventTypeMeetingEnded {
-		w.WriteHeader(http.StatusNotImplemented)
-		return
-	}
+	p.handleStandardWebhook(w, r, &webhook)
 
-	var meetingWebhook zoom.MeetingWebhook
-	err = json.Unmarshal(b, &meetingWebhook)
-	if err != nil {
-		p.API.LogError("Error unmarshaling meeting webhook", "err", err.Error())
-		http.Error(w, err.Error(), http.StatusBadRequest)
+	// TODO: handle recording webhook
+}
+
+func (p *Plugin) handleStandardWebhook(w http.ResponseWriter, r *http.Request, webhook *zoom.Webhook) {
+	if webhook.Status != zoom.WebhookStatusEnded {
 		return
 	}
```

Comment:
0/5 maybe keep the correct header here
```suggestion
	if webhook.Status != zoom.WebhookStatusEnded {
		w.WriteHeader(http.StatusNotImplemented)
		return
	}
```